docs/src/pages/customization/themes/WithTheme.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import { withTheme, Theme } from '@material-ui/core/styles';
+
+function WithTheme(props: { theme: Theme }) {
```

Comment:
`docs/src/pages/customization/themes/WithTheme.js` is dead code. The old JavaScript and this new TypeScript file can be removed :).

Reply:
Same problem with `docs/src/pages/customization/themes/ResponsiveFontSizesChart.js`, to delete.

Reply:
Seems `docs/src/pages/customization/themes/WithTheme.js` isn't dead, it's imported from `docs\src\pages\customization\palette\DarkTheme.js`

Reply:
Oh, that's strange, thanks for spotting it. Can we remove the import, merge the two files?

Reply:
Hum, hold on. Forget about it. We are diverging from the initial effort of the pull request.

Reply:
Files restored